tests/tablewalker.js

```diff
@@ -202,25 +203,25 @@ describe( 'TableWalker', () => {
 				[ '22' ],
 				[ '30', '31', '32' ]
 			], [
-				{ row: 1, column: 0, index: 0, data: undefined },
-				{ row: 1, column: 1, index: 0, data: undefined },
-				{ row: 1, column: 2, index: 0, data: '12' },
-				{ row: 2, column: 0, index: 0, data: undefined },
-				{ row: 2, column: 1, index: 0, data: undefined },
-				{ row: 2, column: 2, index: 0, data: '22' }
+				{ row: 1, column: 0, index: 0, data: '00', isSpanned: true },
+				{ row: 1, column: 1, index: 0, data: '00', isSpanned: true },
+				{ row: 1, column: 2, index: 0, data: '12', isSpanned: false },
+				{ row: 2, column: 0, index: 0, data: '00', isSpanned: true },
```

Comment:
WDYT about making `isSpanned: false` default value in walker test?

If set would stand-out more:
```js
{ row: 1, column: 0, index: 0, data: '00', isSpanned: true },
{ row: 1, column: 1, index: 0, data: '00', isSpanned: true },
{ row: 1, column: 2, index: 0, data: '12' },
{ row: 2, column: 0, index: 0, data: '00', isSpanned: true }
{ row: 2, column: 2, index: 0, data: '22' }
```